lib/WebAssemblyParser.js

```diff
@@ -4,28 +4,40 @@
 */
 "use strict";
 
-const Tools = require("webassembly-interpreter/lib/tools");
-
-const Tapable = require("tapable").Tapable;
+const {
```

Comment:
is multiline required here for single variable declaration by code style configuration?

Reply:
yes